Directory.Build.props

```diff
@@ -11,8 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.113" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8" PrivateAssets="all" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.261" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.1" PrivateAssets="all" />

```

Comment:
Actually, I think we no longer need this package.
These analyzers are included in the .NET 5.0 SDK, with most of the rules disabled by default, but we can enable them with a property like this:
```xml
<AnalysisMode>AllEnabledByDefault</AnalysisMode>
```